mkdocs.yml

```diff
@@ -229,7 +229,9 @@ nav:
       - User interface: user-guide/etendo-mobile/user-interface/user-interface.md
       - Bundles:
         - Etendo Classic Subapp Extensions: user-guide/etendo-mobile/bundles/etendo-classic-subapp-extensions/classic-subapp-extensions.md
-    - Etendo Copilot: 
+    - Etendo Copilot:
+      - Assistants:
```

Comment:
Change the order, first getting started

Reply:
Rename Purchase Assistant, since it is within the copilot section, it is repetitive.